### ISSUE TITLE
Nova Version 10

### ISSUE THREAD

**Issue by motownphilippe** [dennisosaj/advancedphp.novaextension#9]
- Since the update to v10 comment lines in advancedPHP files results in html comments tags not PHP comments. 
- The closing '?>' is often purple, as if it was a quoted string. 
<img width="993" alt="Screenshot 2022-12-03 at 7 51 53 am" src="https://user-images.githubusercontent.com/74905998/205384402-0bd0ea7e-5bbe-40fe-aad7-0cfef6d36c74.png">

**Comment by dennisosaj** [comment]
Yes, I noticed a lot of other things, too. My personal solution was to stay on Nova 9 as long as I find the time to look into it.

**Comment by motownphilippe** [comment]
Thanks again for your work on it. I look forward to the update whenever you get time. Philippe On 6 Dec 2022, at 7:25 pm, Dennis Osaj ***@***.***> wrote:﻿
Yes, I noticed a lot of other things, too. My personal solution was to stay on Nova 9 as long as I find the time to look into it.
